app/controllers/api/v1/ratings_controller.rb

```diff
@@ -11,34 +11,27 @@ def show
   end
 
   def create
-    users_rated_skills = current_user.ratings.map { |r| r.skill }
-    skill = Skill.where(id: params["rating"]["skill"]).first
-    if users_rated_skills.include?(skill)
-      return revise(skill)
-    end
     rating = Rating.new(rating_params)
-    rating.user = User.where(id: session[:user_id]).first
-    rating.skill = Skill.where(id: params["rating"]["skill"]).first
+    rating.user = current_user
+    rating.skill = Skill.find(params["rating"]["skill"])
 
     if rating.save
       render status: 201, json: rating, root: "rating"
     else
-      render status 422, json: { rating: { errors: rating.errors.full_messages } }
+      render status: 422, json: {
+        rating: { errors: rating.errors.full_messages }
+      }
     end
   end
 
-  def revise(skill)
-    rating = current_user.ratings.where(:skill => skill).first
-    rating.score = params["rating"]["score"]
-    render status: 200, json: rating, root: "rating" 
-  end
-
```

Comment:
I believe Luke's(?) thinking here was that when Ember creates a new rating, if it exists already, update it instead. I'm assuming the reasoning for taking it out is that it would be better to do this from the ember side of things?